### PR TITLE
New stratum 0 options for cvmfs 2.2.

### PR DESCRIPTION
**Pull request by traylenator**

### CODE REVIEW

File: manifests/zero.pp [hunk 1]
```diff
@@ -15,7 +15,8 @@
   $garbage_collection = false,
   $auto_gc = false,
   $auto_gc_timespan = '3 days ago',
-  $ignore_xdir_hardlinks = false
+  $ignore_xdir_hardlinks = false,
+  $creator_version = '2.1.19',
```
Comment (reneme) [inline manifests/zero.pp]: shouldn't that be `2.2.0-1`?

Comment (traylenator) [inline manifests/zero.pp]: It's a parameter defaulting to current value. Dan will override it when the time comes.

File: templates/zero-httpd.conf.erb [hunk 2]
```diff
@@ -11,13 +11,16 @@ Alias /cvmfs/<%= @repo %> <%= @repo_store %>/<%= @repo %>
...
+
 
     Header unset Last-Modified
     FileETag None
 
     ExpiresActive On
     ExpiresDefault "access plus 3 days"
-    ExpiresByType text/html "access plus 5 minutes"
     ExpiresByType application/x-cvmfs "access plus 2 minutes"
+    ExpiresByType application/json    "access plus 2 minutes"
```
Comment (reneme) [inline templates/zero-httpd.conf.erb]: `application/json` is not known by SL5, hence we added  `AddType application/json .json` to the [upstream configuration](https://github.com/cvmfs/cvmfs/blob/b4f4297e54f1b65c31a79f78ee52a719ca466c65/cvmfs/cvmfs_server#L2018) just before the `<Directory ...>` block.

File: manifests/zero.pp (hunk 1 above)
Comment (reneme) [inline manifests/zero.pp]: d'accord.

File: templates/zero-httpd.conf.erb (hunk 2 above)
Comment (traylenator) [inline templates/zero-httpd.conf.erb]: SL5 is a disant memory as far as I'm concerned.

**Pull request merged**